### DISCUSSION TITLE
NModal only content scrollable

### DISCUSSION THREAD

**Discussion by TheFreeman** [tusen-ai/naive-ui#4292] [Ideas]
Hi,

currently the NModal enables scrolling only over the complete height.
This is no good usability. The header and the footer are also scrolling.
On long content, it feels not really good.
Is there any possibility to implement a property for to set scrollbar-behavior only for the content-slot on NCard-behavior?
This would be a great and good feature! :-)

Kind regards,
TF
------------- deepl.com -------------
你好。

目前，NModal只允许在整个高度上滚动。
这不是很好的可用性。页眉和页脚也是滚动的。
对于长的内容，感觉不是很好。
有没有可能实现一个属性，只为NCard-behavior上的内容槽设置滚动条行为？
这将是一个很好的功能。:-)

好心的问候。
尊敬的先生

**Comment by MaxMueller0815** [comment]
I would also appreciate it

**Comment by freifunk-ingolstadt** [comment]
sounds good
